heartexlabs/label-studio
# docs: Remove HumanSignal changelog from releases notes for now

### Describe the reason for change
Changelog is not available on humansignal.com at the moment

### How to test this

1. Visit https://deploy-preview-4450--heartex-docs.netlify.app/guide/release_notes.html
2. The admonition for the enterprise changelog should be gone
